src/app/_core/constants/pagination.constants.ts

```diff
@@ -1,4 +1,4 @@
-export const limitParam = 12;
-export const pageParam = 0;
-export const pokemonNumber = 1025;
-export const pageSizeOptions = [12, 24, 48, 72];
+export const limitParam: number = 12;
+export const pageParam: number = 0;
```

Comment:
pageParam is misleading name. It represents actually the  DEFAULT_PAGE. But when reading the name you think it is the queryParam key .

Always try to find the best and most explicit names for your variables and function. It's really important for code clarity